documentation/src/main/asciidoc/user_guide/chapter-48-Using_Hot_Rod_Server.adoc

```diff
@@ -1271,20 +1271,12 @@ is detailed in the link:#_configuration_7[Querying Infinispan] chapter.
 There is however an extra configuration step involved. While in embedded mode the indexing metadata is obtained via Java
 reflection by analyzing the presence of various Hibernate Search annotations on the entry's class, this is obviously not
 possible if the entry is protobuf encoded. The server needs to extract the relevant metadata from the same descriptor
-(.proto file) as the client. The descriptor is currently supplied to the server by remotely invoking the
-_ProtobufMetadataManager_ MBean via JMX. The _ProtobufMetadataManager_ is a cluster-wide replicated repository of
-protobuf descriptors. For each running cache manager a separate _ProtobufMetadataManager_ MBean instance exists. Its
-ObjectName follows this pattern:
-
- <jmx domain>:type=RemoteQuery,name=<cache manager name>,component=ProtobufMetadataManager
-
-The method relevant for registering the protobuf descriptor file has the signature:
-
- void registerProtofile(String names[]. String contents[])
-
-A detailed example for all the described steps can be seen in the Java Hot Rod client test suite, particularly in
-link:https://github.com/infinispan/infinispan/blob/master/client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/HotRodQueryTest.java[HotRodQueryTest]
-
+(.proto file) as the client. The descriptors are stored in a dedicated cache on the server _'___protobuf_metadata'_.
+Registering a new schema is therefore as simple as performing a _put_ operation on that cache using the schema's name as
+a key and the schema itself as the value. Alternatively you can use the CLI (via the
+cache-container=*:register-proto-schemas() operation), the Console or the _ProtobufMetadataManager_
+MBean via JMX. Be aware that, when security is enabled, access to the schema cache via the remote protocols requires
+that the user belongs to the _'___schema_manager role'_
```

Comment:
@tristantarrant Could you put each sentence on a separate line? It will require less re-wrapping for future changes.